src/Miunie.Core/Services/TimeService.cs

```diff
@@ -30,6 +30,50 @@ public async Task OutputCurrentTimeForUserAsync(MiunieUser user, MiunieChannel c
             await _messages.SendMessageAsync(channel, PhraseKey.TIME_TIMEZONE_INFO, user.Name, targetDateTime);
         }
 
+        public async Task OutputCurrentTimeComparedToInputForUserAsync(MiunieUser requestUser, DateTime requestTime, MiunieUser user, MiunieChannel channel)
+        {
+            var requesterOffset = requestUser.UtcTimeOffset ?? new TimeSpan();
+            var otherUserOffSet = user.UtcTimeOffset ?? new TimeSpan();
+
+            var requestUtcTime = requestTime - requesterOffset;
+            var otherUserTime = requestUtcTime + otherUserOffSet;
+
+            var formattedRequestTIme = requestTime.ToShortTimeString();
+            var formattedOtherUserTime = otherUserTime.ToShortTimeString();
+
+            await _messages.SendMessageAsync(channel, PhraseKey.TIME_USERTIME_FROM_LOCAL, formattedRequestTIme, user.Name, formattedOtherUserTime);
+        }
+
+        public async Task OutputMessageTimeAsLocalAsync(ulong messageId, DateTimeOffset? createdTimeOffset, DateTimeOffset? editTimeOffset, MiunieUser user, MiunieChannel channel)
+        {
+            if (createdTimeOffset == null)
```

Comment:
I'd say let's use the `is` operator as a rule of thumb.

I just quickly found [this article](https://www.gullberg.tk/blog/is-null-versus-null-in-c/) as an example why we might want to use `is` instead of `==`.

There is no real pressing issue with `==`, it's just a matter of keeping up with the new standards.

```suggestion
            if (createdTimeOffset is null)
```